src/context.rs

```diff
@@ -68,11 +68,14 @@ pub const DIRECTIONAL_MODES: usize = 8;
 pub const KF_MODE_CONTEXTS: usize = 5;
 
 pub const EXT_PARTITION_TYPES: usize = 10;
-pub const TX_SIZES: usize = 4;
+
+//pub const TX_SIZES: usize = TxSize::TX_SIZES;
+pub const EXT_TX_SIZES: usize = 4;
```

Comment:
We used to use the original name as in libaom, 'EXT_TX_SIZES' but renamed it to TX_SIZES.
Rav1e don't need to use the same name, and further more av1 spec does not use any 'EXT_', 
simply because all those 'EXT_' is default in av1.
Similar changes have applied to other variable or function names having 'EXT' or 'ext', I remember.

Reply:
Is having two constants called TX_SIZES with different values desirable, or would it make sense to put the second TX_SIZES in a more descriptive namespace (or something like that)?

Reply:
Having two different definition for TX_SIZES is clearly wrong. Also, its value should be 5 as you probably stated. I briefly checked  that where TX_SIZES defined in context.rs is used (It is static so only can be used in that file), then several multi-dim cdf definition referenced it and puzzled how could they worked correctly so far (Because all those cdfs are imported from libaom as extern .h or .c!
Btw, there is one reason why TX_SIZES = 4 has been used. That is, Rav1e has not enabled TX_64X64. So the tx size index 4 has nowhere been used in the code, as index to any array or etc.

Reply:
In that case it seems like there should be more data for some of these cdf tables, since I pulled these straight from the C code and using a value of 5 does not work. The C code uses EXT_TX_SIZES which is 4 (and has the second TX_SIZES = 5 used in other places)  Does it make sense to 0-fill the tables as necessary in case at some point those values become available?

Reply:
I think I'm beginning to see how EXT_TX_SIZES is being used with relation to the intra/inter tx sets.  I will take another run at it in the morning.